Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "deepgram"
-version = "0.6.6"
+version = "0.6.7"
```

Comment:
Let's bump to 0.7.0!

Reply:
Actually, I think @jcdyer  does this in separate PRs when doing a public release. I'll revert it.